src/Symfony/Component/Form/Tests/AbstractBootstrap4HorizontalLayoutTest.php

```diff
@@ -53,7 +53,7 @@ public function testLabelOnForm()
 
         $this->assertMatchesXpath($html,
 '/legend
-    [@class="col-form-label col-sm-2 col-form-legend required"]
+    [@class="col-form-label col-sm-2 col-form-label required"]
```

Comment:
Here is the `col-form-label` added twice. Do we bother?

Reply:
it would be better if we could avoid, to make the HTML cleaner. But it should not break anything.